### PR TITLE
12.0 barcodes generator product fix show fields in product template

### PR DESCRIPTION
**Pull request by cvinh**
Barcode fields and buttons should be invisible if product_template has variants
Wait for https://github.com/OCA/stock-logistics-barcode/pull/201 to be merged

### CODE REVIEW

**Comment by rousseldenis** [comment]
@cvinh Could you add in PR first message that we wait for #201 to be merged ?

**Comment by pedrobaeza** [comment]
Please rebase after #201 gets merged

**Comment by rousseldenis** [comment]
@cvinh

File: barcodes_generator_product/models/product_template.py
Comment (legalsylvain) [inline barcodes_generator_product/models/product_template.py]: Hi. Why did you add readonly= False. Is it not the default value ?

Comment (rousseldenis) [reply barcodes_generator_product/models/product_template.py]: @legalsylvain On v >= 12, related fields are readonly=True by default

Comment (legalsylvain) [reply barcodes_generator_product/models/product_template.py]: oh thanks !

**Comment by rousseldenis** [comment]
@cvinh I think your rebase is incorrect

**Review by legalsylvain — CHANGES_REQUESTED** [review]
114 files changed. I guess something is wrong in your rebase. don't you think ?

**Comment by rousseldenis** [comment]
@cvinh

**Comment by cvinh** [comment]
> @cvinh

done

**Comment by cvinh** [comment]
Hello, is there a psc member who can apply 'no stale' please ?

**Review by rousseldenis — APPROVED** [review]
Seems legit

**Comment by dreispt** [comment]
Merged with #612

**Pull request closed without merging**
